ocs_ci/utility/utils.py

```diff
@@ -2054,3 +2055,84 @@ def convert_device_size(unformatted_size, units_to_covert_to):
             return absolute_size / 1024
         elif units == 'Mi':
             return absolute_size
+
+
+def mirror_image(image):
+    """
+    Mirror image to mirror image registry.
+
+    Args:
+        image (str): image to be mirrored, can be defined just with name or
+                     with full url, with or without tag or digest
```

Comment:
here should be only one indent from above line starting with `image (str)`

Reply:
Fixed in 43884ad.